projects/packages/search/src/dashboard/components/record-meter/notice-box.jsx

```diff
@@ -19,6 +19,10 @@ const CLOSE_TO_LIMIT_PERCENT = 0.8; //TODO: currently 'close' is defined as 80%.
  */
 export function NoticeBox( props ) {
 	const notices = [];
+	const record_limit_as_string =
```

Comment:
I think we want `recordLimitAsString` to match the variable naming convention.

Reply:
ah, nice catch. thanks!